src/app/helptext/task-calendar/replication.ts

```diff
@@ -0,0 +1,155 @@
+import { T } from '../../translate-marker';
+
+export default {
+    name_placeholder: T('Name'),
+    name_tooltip: T('Replication task name'),
+
+    direction_placeholder: T('Direction'),
+    direction_tooltip: T('Direct the flow of the ZFS snapshots. Choose between pushing\
```

Comment:
'Direction of travel. <i>Push</i> sends snapshots to a remote system. <i>Pull</i> receives snapshots from a remote system.'